bio-qiime-pipeline/src/import_multiplexed.py

```diff
@@ -0,0 +1,132 @@
+from typing import Optional, List
+from pathlib import Path
+from zipfile import ZipFile
+
+import logging
+
+from coretex import CustomDataset, CustomSample, Experiment, folder_manager
+from coretex.bioinformatics import ctx_qiime2
+
+from .utils import convertMetadata
+from .caching import getCacheNameOne
+
+
+FORWARD_FASTQ = "forward.fastq"
+REVERSE_FASTQ = "reverse.fastq"
+BARCODES_FASTQ = "barcodes.fastq"
+
+
+def importSample(sequencesPath: Path, sequenceType: str, outputDir: Path) -> Path:
+    importedSequencesPath = outputDir / "multiplexed-sequences.qza"
+
+    ctx_qiime2.toolsImport(sequenceType, str(sequencesPath), str(importedSequencesPath))
+
+    outputPath = outputDir / "import-output.zip"
+    with ZipFile(outputPath, "w") as outputFile:
+        outputFile.write(importedSequencesPath, importedSequencesPath.name)
+
+    return outputPath
+
+
+def importMetadata(metadataPath: Path, outputDir: Path) -> Path:
+    metadataPath = convertMetadata(metadataPath)
+
+    outputPath = outputDir / "metadata-output.zip"
+    with ZipFile(outputPath, "w") as outputFile:
+        outputFile.write(metadataPath, metadataPath.name)
+
+    return outputPath
+
+
+def getFastq(sample: CustomSample, fileName: str) -> Optional[Path]:
+    foundFiles = list(sample.path.glob(f"*{fileName}"))
```

Comment:
instead of using `glob` use `sample.joinPath` and compress file if it is not already gz compressed